versioned_docs/version-8.7/self-managed/operational-guides/production-guide/helm-chart-production-guide.md

```diff
@@ -53,8 +53,6 @@ kubectl create namespace orchestration
 
 - **Namespace `management`:** We will install [Identity](/self-managed/identity/what-is-identity.md), [Console](/self-managed/console-deployment/overview.md), and all the [Web Modeler](/self-managed/modeler/web-modeler/installation.md) components.
 
-- **Namespace `orchestration`**: We will install [Zeebe](/self-managed/zeebe-deployment/zeebe-installation.md), the Camunda web applications ([Operate](/self-managed/operate-deployment/install-and-start.md), [Tasklist](/self-managed/tasklist-deployment/install-and-start.md), and [Optimize](/$optimize$/self-managed/optimize-deployment/install-and-start.md)), along with [Connectors](/self-managed/connectors-deployment/install-and-start.md).
```

Comment:
This is probably not intended, let's revert the change.

Reply:
Oh, I had to comment this out because the docs wouldn't compile on my machine. 🌚